test/1_test_fullpath.js

```diff
@@ -547,24 +559,18 @@ contract('Voucher tests', async (addresses) => {
 
       //Check BosonRouter state
       assert.equal(
-        await contractBosonRouter.correlationIds(users.buyer.address),
+        await contractBosonRouter.getCorrelationId(users.buyer.address),
         1,
         'Correlation Id incorrect'
       );
 
       //Check Voucher Kernel state
-      const voucherStatus = await contractVoucherKernel.vouchersStatus(
+      const voucherStatus = await contractVoucherKernel.getVoucherStatus(
         tokenVoucherKey
       );
-      assert.isTrue(voucherStatus.status.eq(new BN(128))); //128 = COMMITTED
-      assert.isFalse(
-        voucherStatus.isPaymentReleased,
-        'Payment released not false'
-      );
-      assert.isFalse(
-        voucherStatus.isDepositsReleased,
-        'Deposit released not false'
-      );
+      assert.isTrue(voucherStatus[0].eq(new BN(128))); //128 = COMMITTED
```

Comment:
Please use constants instead of hard-coded indexes for voucherStatus (same as remark above)